src/http/server.js

```diff
@@ -2,14 +2,26 @@ const http = require('http');
 const url = require('url');
 const logger = require('winston');
 
-http.createServer((req, res) => {
-    const urlparameters = url.parse(req.url, true);
-    // TODO: make path read from config
-    if (urlparameters.pathname === '/easyCalAuth') {
-        const token = urlparameters.query.code;
-        const guild = urlparameters.query.state;
-        logger.info(token);
-        logger.info(guild);
+let lport = 8080;
+
+module.exports.setPort = (port) => {
+    if (typeof port === 'number' && port <= 65535) {
+        lport = port;
+    } else {
+        throw new RangeError('Invlaid Listening Port');
```

Comment:
Fix the typo here and maybe also log the port, something like ``RangeError(`Invalid listening port ${port}`); ``